lib/MultiCompiler.js

```diff
@@ -15,8 +15,10 @@ module.exports = class MultiCompiler extends Tapable {
 		super();
 		this.hooks = {
 			done: new SyncHook(["stats"]),
-			invalid: new SyncHook([]),
-			watchClose: new SyncHook([])
+			invalid: new SyncHook(["filename", "changeTime"]),
```

Comment:
replace `invalid` `watchClose` `run` and `watchRun` with `tapable.MultiHook` which delegrates to the passed hooks.

Example: `invalid: new MultiHook(compilers.map(c => c.hooks.invalid))`

Reply:
Keep `done` as it's now.